guidelines/responses.md

```diff
@@ -61,11 +61,32 @@ See the [Error codes](#error-codes) table below for further details about specif
 }
 ```
 
+### Example \#4
+
+```javascript
+{
+   "success": false,
+   "errors":[
+      {
+         "code":9,
+         "message":"Invalid rate code",
+         "rateCode":"ABC"
+      },
+      {
+         "code":10,
+         "message":"Invalid category code",
+         "categoryCode":"XYZ"
+      }
+   ]
+}
+```
+
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `success` | `bool` | required | Determinines the result of the operation. |
 | `asyncConfirmation` | `bool` | optional | Determinines if there will be following asynchronous response. |
-| `errors` | array of [`Error`](#error) | optional | In case of `"success": false`, this property holds information about the errors that occurred. |
+| `error` | [`Error`](#error) | optional | In case of `"success": false`, this property holds information about the error that occurred. The singular form `error` should be used for a single error. |
```

Comment:
@patrikmiksovsky95 single error in response is still supported? or is it rather deprecated? the doc examples (all 3 of them currently) show it as single item

![image](https://github.com/MewsSystems/gitbook-channel-manager-api/assets/773595/61fbdb77-4c45-4ec3-8d55-2cbf971489b9)

Reply:
rather deprecated

Reply:
Just to be clear, are we therefore saying that `error` as a single object is technically supported but **deprecated**, so API users should instead always be using the plural form `errors` and this should always be an array, even if there is a single error? So, for example, ```"error": [{"code":10, "message":"It ain't working"}]```?

Reply:
@MikeAdamsMews yep

Reply:
Thanks for the clarification @ondrahermanek, in that case I need to edit my changes to (i) mark 'error' (singular) as deprecated; (ii) remove 'error' (singular) from the examples and replace it with one for a singular error in 'errors' (plural).